src/blocks/HeaderBlockBrandPivot/Banner.tsx

```diff
@@ -0,0 +1,33 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {
+  MinimalColorComponent,
+  MarkdownHtmlComponent,
+} from 'src/blocks/BaseBlockProps'
+import { SectionWrapper, TABLET_BP_UP } from '../../components/blockHelpers'
+
+const BannerContent = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0.5rem;
+  text-align: center;
+  font-size: 0.875rem;
+
+  ${TABLET_BP_UP} {
+    font-size: 1rem;
+  }
+
```

Comment:
Actually, I think just setting this a white-space rule on all links fix the issues and we can still keep the flexibility of the markdown field :) 
```suggestion
a {
  white-space: nowrap;
}
```